quesma/model/bucket_aggregations/date_histogram.go

```diff
@@ -17,7 +18,7 @@ import (
 type DateHistogramIntervalType bool
 
 const (
-	DefaultMinDocCount                                      = 1
+	DefaultMinDocCount                                      = -1
```

Comment:
Why? It's confusing to me, as the default seems to be `1`

Reply:
Now the default is to return all buckets from first hit till last one even if they are zeros. A lot of tests break b/c of that.